projects/packages/my-jetpack/_inc/components/products-table-view/index.tsx

```diff
@@ -63,6 +70,29 @@ const compileData: (
 	return data;
 };
 
+const getCategories: (
+	products: JetpackModule[],
+	allProductData: {
+		[ key: string ]: ProductCamelCase;
+	}
+) => Option[] = ( products, allProductData ) => {
+	const categories = [
+		...new Set(
+			products.map( product => {
+				const productData = allProductData[ product ];
+				return productData.category;
+			} )
+		),
+	];
+
+	const categoryOptions = categories.map( category => ( {
+		value: category,
+		label: category.charAt( 0 ).toUpperCase() + category.slice( 1 ),
```

Comment:
I would say we could use `text-transform: uppercase` wherever this is being displayed lol

Reply:
I forgot this CSS property existed, thank you!

Reply:
and I have no idea why I sad `uppercase` instead of `capitalize` 😂 😂 😂

Reply:
No worries, I knew what you meant 😆